packages/clerk-js/src/ui/elements/Select.tsx

```diff
@@ -309,16 +311,13 @@ export const SelectButton = (props: PropsOfComponent<typeof Button>) => {
     <Button
       ref={reference}
       colorScheme='neutral'
-      variant='ghost'
+      variant='select'
```

Comment:
❓ Do we need a variant for that? It looks like we're only using it in a single place. Do you think we can move the corresponding styles inside the Select component directly for now?

Reply:
Not right now. I guess we can use the `ghost` variant which uses the same properties from `vars` and override the other properties in the `SelectButton` for now.